### ISSUE TITLE
LCD powers on but Pi doesn't boot

### ISSUE THREAD

**Issue by davidpic76** [kiteretro/Circuit-Sword#74]
I have two Circuit Swords that I have just started testing.
One boots up perfectly but the other one doesn't boot but the LCD powers on.
I switched the compute module, screen and SD card from the working one and got the same result so I think its the board that has the issue.

Any ideas?

![img_7798](https://user-images.githubusercontent.com/36372143/47584156-47130b00-d90e-11e8-8c40-06481c82cb10.jpg)

**Comment by davidpic76** [comment]
Rotated the image!

![img_7798](https://user-images.githubusercontent.com/36372143/47584193-67db6080-d90e-11e8-9eb2-8b995fab2ce0.jpg)

**Comment by davidpic76** [comment]
This looks like a dupe of https://github.com/kiteretro/Circuit-Sword/issues/69

**Comment by kiteretro** [comment]
Indeed, sorry about this, there is a ‘known issue’ already, could you send me an email to kite@kitesitemshop.com and include your full address and I’ll sort a replacement and give you a return address (and I’ll pay for the return shipping back to me)

**Comment by kiteretro** [comment]
Oh looks like you already have ;)

**Comment by davidpic76** [comment]
Haha no problem

My address is

David Pickford
2045 Jonathan Ave
San Jose, CA, 95125
USA


On Oct 26, 2018, at 2:15 PM, kiteretro <notifications@github.com<mailto:notifications@github.com>> wrote:

Oh looks like you already have ;)
